### DISCUSSION TITLE
Touch Monitor (Virtual)

### DISCUSSION THREAD

**Discussion by fernando-inf** [MolotovCherry/virtual-display-rs#137] [Q&A] [answered]
Hello, can this program simulate a Touch Monitor? Currently I cannot control my Windows 10 by touch when I log in from my Android tablet (I´m use Anydesk), I can only move the cursor like a touchpad.

**Comment by MolotovCherry** [comment, marked as answer]
Hi, no, this does not simulate a touch screen device~
